omod/src/main/webapp/fragments/dashboard.gsp

```diff
@@ -0,0 +1,155 @@
+<%
+    ui.includeJavascript("financials", "jquery.dataTables.min.js")
+    ui.includeCss("financials", "jquery.dataTables.min.css")
+    ui.includeCss("financials", "bootstrap.min.css")
+    ui.includeCss("financials", "main.css")
+%>
+
+<link rel="stylesheet" type="text/css" href="./bootstrap.min.css">
+<link rel="stylesheet" type="text/css" href="./bootstrap-datetimepicker.css">
+<link rel="stylesheet" type="text/css" href="./main.css">
+
+<link rel="stylesheet" type="text/css" href="./css/bootstrap.min.css">
+<script type="text/javascript" src="./js/bootstrap.min.js"></script>
+<script type="text/javascript" src="./js/bootstrap-datetimepicker.js"></script>
+<script src="https://cdn.anychart.com/releases/8.0.0/js/anychart-base.min.js"></script>
```

Comment:
Accessing an online resource here means that it will NOT work for those running the application locally, if you have a library that you want to use, you first check in the `ehrconfigs` if it is missing, it is better to download it and include it there, so that other implementations can have a central repo for resources

Reply:
Also resource access should be accessed per the UIFramework standards between the <% file%> as done in the header, also all those files might be in the ehrconfigs

Reply:
let me check out how to fix this